### PR TITLE
Networking response

### PR DESCRIPTION
**Pull request by martin371998**
Fixed networking layer response handling

### CODE REVIEW

File: app/src/main/java/com/example/euriskocodechallenge/ui/home/news/NewsRVAdapter.kt [hunk 1]
```diff
@@ -48,10 +50,13 @@ class NewsRVAdapter : RecyclerView.Adapter<NewsRVAdapter.NewsViewHolder>() {
         holder.binding.apply {
             titleTv.text = newsList[position].title
             authorTv.text = newsList[position].byline
-            newsThumb.load(newsList[position].media[0].mediaMetadata[0].url) {
-                crossfade(true)
-                crossfade(1000)
-                transformations(RoundedCornersTransformation(10F))
+            if(newsList[position].media[0].mediaMetadata[0].url.isNullOrEmpty()){
```
Comment (joe-abouassi) [inline app/src/main/java/com/example/euriskocodechallenge/ui/home/news/NewsRVAdapter.kt]: @martin371998 already commented in another PR

**Review by joe-abouassi — APPROVED** [review]

File: app/src/main/java/com/example/euriskocodechallenge/ui/home/news/NewsRVAdapter.kt (hunk 1 above)
Comment (martin371998) [reply app/src/main/java/com/example/euriskocodechallenge/ui/home/news/NewsRVAdapter.kt]: done in another branch

File: app/src/main/java/com/example/euriskocodechallenge/ui/login/SignUpActivity.kt [hunk 2]
```diff
@@ -30,37 +31,47 @@ class SignUpActivity : AppCompatActivity() {
...
         userViewModel.fetchUser().observe(this) {
-            Toast.makeText(this, "Welcome ${it.fName} ${it.lName}", Toast.LENGTH_LONG).show()
+            utilityfunctions.showtoast(this, "Welcome ${it.fName} ${it.lName}")
             startActivity(Intent(this, HomeActivity::class.java))
         }
         //- Can Be Removed - Checks if user inserted successfully
         userViewModel.fetchInsertedId().observe(this) {
             if (it != -1L && it != 0L) {
-                Toast.makeText(this, "Registration Successful", Toast.LENGTH_LONG).show()
+                utilityfunctions.showtoast(this, "Registration Successful")
```
Comment (joe-abouassi) [inline app/src/main/java/com/example/euriskocodechallenge/ui/login/SignUpActivity.kt:72]: @martin371998 hardcoded strings should be in strings.xml file

**Review by joe-abouassi — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/euriskocodechallenge/ui/login/SignUpActivity.kt (hunk 2 above)
Comment (martin371998) [reply app/src/main/java/com/example/euriskocodechallenge/ui/login/SignUpActivity.kt:72]: fixed in Utility functions new PR

**Review by joe-abouassi — APPROVED** [review]

**Pull request merged**